crates/router/src/core/user/dashboard_metadata.rs

```diff
@@ -548,6 +556,17 @@ async fn insert_metadata(
             )
             .await
         }
+        types::MetaData::OnboardingSurvey(data) => {
+            utils::insert_merchant_scoped_metadata_to_db(
```

Comment:
The enum will be one time set, not updatable?

Reply:
Adressed